docroot/modules/custom/mass_content/src/EventManager.php

```diff
@@ -82,10 +97,9 @@ private function getPastQuery(NodeInterface $parent) {
    */
   private function getUpcomingQuery(NodeInterface $parent) {
     $query = $this->getBaseQuery($parent);
-    $today = new \DateTime('today', new \DateTimezone('America/New_York'));
-    $today->setTimezone(new \DateTimezone('UTC'));
-
-    $query->condition('field_event_date.end_value', $today->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT), '>');
+    $upcomingDateEndDate = new \DateTime($this->upcomingEndDateRelative, new \DateTimezone('America/New_York'));
```

Comment:
Should this be `DrupalDateTime`?

Should we use the site timezone instead of hardcoding America/New_York?

Reply:
I will change the references of DateTimezone to DrupalDateTime. Great suggestions as using DrupalDateTime there is no need to set the timezone when using relative dates. Thanks!